### PR TITLE
APIN Team Push

### PR DESCRIPTION
**Pull request by wonderfulguru1**
New Commits from APIN Team

### CODE REVIEW

**Comment by wonderfulguru1** [comment]
These lines were actually commented out before I pull from the repo the. My only addition was to add the parameter "true" to the get patients service at line 93. This will ensure that the service pulls all patients.

File: api/src/main/resources/liquibase.xml [hunk 1]
```diff
@@ -35,5 +35,32 @@
         <addForeignKeyConstraint constraintName="nigeriaemr_item_owner_fk" baseTableName="nigeriaemr_item" baseColumnNames="owner"  referencedTableName="users"
                                  referencedColumnNames="user_id" />
     </changeSet> -->
+
```
Comment (wandechris) [inline api/src/main/resources/liquibase.xml:38]: kudos for using liquidbase for DB migrations

**Comment by talk2morris** [comment]
Hello @wandechris I had to request a review because I noticed this pushed ought to have up the version to 1.6. @wonderfulguru1  is aware of this.

**Comment by wandechris** [comment]
> Hello @wandechris I had to request a review because I noticed this pushed ought to have up the version to 1.6. @wonderfulguru1 is aware of this.

@talk2morris do you mean 1.4.3-SNAPSHOT?

**Comment by wonderfulguru1** [comment]
Updated new plugin version from 1.4.1 to 1.6.0 version

**Comment by talk2morris** [comment]
Hi @wonderfulguru1 Well done on this..Although you missed updating the pom for the api module and the parent module, this would cause conflict. kindly adjust this.

**Pull request merged**